\documentclass[12pt, letterpaper]{article}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{fancyhdr}
\usepackage{geometry}
%\usepackage{natbib}
\usepackage{indentfirst} % to indent the first paragraph after a section heading\
\usepackage{tabularray}
\usepackage{xcolor}

\usepackage{apacite}
\bibliographystyle{apacite-copy}
%\setcitestyle{aysep={,}} % separate author name and year with a comma
\renewcommand{\BRetrievedFrom}{" "}
\renewcommand{\BRetrieved}{" "}
\renewcommand{\doiprefix}{https://doi.org/}
\AtBeginEnvironment{thebibliography}{\linespread{1.5}\selectfont} % make bibliography single-spaced

% new command to make a better type of column
\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\arraybackslash}p{#1}}

\usepackage{booktabs, tabularx, longtable}
\usepackage{array}
\usepackage{colortbl}
\usepackage{graphicx} 
\usepackage[section]{placeins}
\usepackage{amsmath}
\usepackage[leftcaption]{sidecap}
\graphicspath{ {./figures/} }
\usepackage{setspace}
\usepackage{longtable}
\renewcommand\arraystretch{1.0}
\usepackage{multirow}

\usepackage{caption}
\captionsetup[table]{font={stretch=1.05, normalfont}}   %% change table caption spacing
\captionsetup[figure]{font={stretch=1.05, normalfont}} %% change figure caption spacing
\usepackage{titlesec}
\titleformat{\section}
 {\normalfont\fontsize{14}{15}\bfseries}{\thesection}{1em}{} %change the size of section header font
\titleformat{\subsection}
 {\normalfont\fontsize{12}{15}\bfseries}{\thesubsection}{1em}{} %change the size of subsection header font
\renewcommand{\thesection}{} %remove the section numbers
\renewcommand{\thesubsection}{} %remove subsection numbers

%try to deal with the problem of too long figure caption
%\DeclareCaptionLabelFormat{adja-page}{\hrulefill\\#1 #2 \emph{(previous page)}}}
%\usepackage{subfig}

%try to deal with issue of figures going to the end of the document
%\makeatletter
%\AtBeginDocument{%
 %\expandafter\renewcommand\expandafter\subsection\expandafter{%
 % \expandafter\@fb@secFB\subsection
 %}
%}
\makeatother

\geometry{margin = 1in}
\pagestyle{fancy}
\fancyhf{}
%set the header
\rhead{\thepage}
\lhead{\textit{O. coloradensis} population dynamics}
%set the line spacing
\renewcommand{\baselinestretch}{2}
%add line numbers
\usepackage{lineno}
\linenumbers

%% Begin the document! 
\begin{document}

\begin{flushleft}
\textbf{Negative density dependence promotes persistence of a globally rare yet locally abundant plant species \textit{(Oenothera coloradensis)}}

\end{flushleft}
\begin{abstract}
\begin{enumerate}

\item Identifying the mechanisms underlying the persistence of rare species has long been a motivating question for ecologists. Classical theory implies that community dynamics should be driven by common species, and that natural selection should not allow small populations of rare species to persist. Yet, a majority of the species found on Earth are rare. Consequently, several mechanisms have been proposed to explain their persistence, including negative density dependence, demographic compensation, vital rate buffering, asynchronous responses of subpopulations to environmental heterogeneity, and fine-scale source-sink dynamics. Persistence of seeds in a seed bank, which is often ignored in models of population dynamics, can also buffer small populations against collapse.  
\item We used integral projection models (IPMs) to examine the population dynamics of \textit{Oenothera coloradensis}, a rare, monocarpic perennial forb, and determine whether any of five proposed demographic mechanisms for rare species persistence contribute to the long-term viability of two populations. We also evaluate how including a discrete seed bank stage changes population models for this species.
\item Including a seed bank stage in population models had a significant positive impact on modeled \textit{O. coloradensis} population growth rate. Using this structured population model, we found that negative density-dependence was the only supported mechanism for the persistence of this rare species. 
\item \textit{Synthesis}: IPMs of two populations of the rare species \textit{O. coloradensis} emphasize the importance of including cryptic life stages such as seed banks in demographic models, but fail to provide strong support for most of the proposed mechanisms of rare species persistence. We propose that high population abundance in micro-site abundances within a spatially heterogeneous environment enables this species to persist, allowing it to sidestep the demographic and genetic challenges of small population size that rare species typically face. The five mechanisms of rare species persistence explored in our study have been demonstrated as effective strategies in other species, and the fact that only one of them had strong support here provides support for the idea that globally rare species can employ distinct strategies for persistence. This reinforces the need for  customized management and conservation strategies that mirror the diversity of mechanisms that allow rare species persistence.  
\end{enumerate}
\end{abstract}

\section{Introduction}
Determining how and why populations of rare species persist has been a goal for ecologists since the discipline's inception \cite{Levins1971RegionalSpecies, Drury1974RareSpecies}. Theoretically, low population size is a final step on a trajectory toward extinction \cite{Stanley1979Macroevolution:Process, Rosenzweig1997TheRarity} or the first step toward ubiquity \cite{Spear2021}.  Yet, small but stable populations of rare species exist in every ecosystem and taxonomic group \cite{Magurran2011CommonnessRarity}. In fact, a large proportion of species globally – as many as 35\% of plant species, for example— can be considered naturally rare \cite{Enquist2019ThePlants}. The prevalence of rarity suggests it is an evolutionarily stable strategy rather than a stop along the path toward extinction or invasion, and implies that there must be both fundamental and realized niches that are available for rare species to occupy. A growing body of evidence demonstrates the importance of rare species for biological processes, including their impacts on community stability \cite{Arnoldi2019ThePatterns, Saterberg2019ADynamics}, and functional composition \cite{Leitao2016RareAssemblages, Burner2022FunctionalSpecies}, which in turn impact ecosystem function \cite{Lyons2005RareFunctioning}. 

Effective conservation and management of rare species require an understanding of both the conditions causing rarity initially, and the mechanisms that allow rare species to persist. Causes of rarity can vary from highly-specific habitat requirements \cite{Sgarbi2018YouTime}, to adverse impacts of anthropogenic environmental change \cite{Vincent2020-tz}. In order to then persist in a state of rarity, a species must overcome any of multiple potential challenges, primarily the negative effects of demographic, environmental, and genetic stochasticity, defined as random variation in vital rates (e.g., survival, reproduction), abiotic conditions, or genetic allele frequencies \cite{May1973StabilityEnvironments}. Stochastic deleterious events can cause extirpation or even extinction of rare species, since there may not be enough unaffected individuals or subpopulations to “rescue” the affected population \cite{Nei1975ThePopulations}. Rare species that maintain populations over time typically do so by employing demographic strategies that compensate for the adverse effects of small population size. There are five main strategies that allow persistence of rare populations (Fig. \ref{fig:conceptualFigure}) \cite{Dibner2019}: negative density-dependence \cite{Rovere2019PersistentlyAvoid}, demographic compensation \cite{Villellas2015DemographicImplications}, vital rate buffering \cite{Pfister1998PatternsImplications, Hilde2020TheChallenges}, asynchronous responses between subpopulations \cite{Abbott2017PortfolioWeberi}, and fine-scale source-sink dynamics \cite{Kauffman2004SpatialPopulation, Pulliam2016SourcesUse}. Negative density-dependence occurs when the growth rate ($\lambda$)  of a population increases at small population size. Demographic compensation occurs when different vital rates are affected in opposing ways by the same perturbation in the environment, which can help maintain a relatively constant population $\lambda$ in response to environmental variation. Vital rate buffering occurs when the variability of vital rates decreases as the vital rate becomes more important (i.e., has a higher elasticity) for population growth rate, which prevents the negative effects of temporal variation on the deterministic $\lambda$ across time \cite{Tuljapurkar1989An.}. Spatial asynchrony occurs when subpopulations close to one another have different or even opposing growth rates, resulting in a stable population-wide $\lambda$. Fine-scale source-sink dynamics occur when there is gene flow between subpopulations that bolsters the size and genetic diversity of very small subpopulations, which again results in a stable population-level $\lambda$. Each of these mechanisms can act independently, but also can interact or overlap \cite{Dibner2019}.  

Here, we identify which of these mechanisms contribute to the persistence or population growth of a rare, endemic plant species, \textit{Oenothera coloradensis} (Rydberg) W.L. Wagner \& Hoch (Onagraceae). We use integral projection models (IPMs) \cite{Easterling2000} that include a discrete seed bank population state. IPMs are flexible models of population dynamics that are constructed using regression models that describe vital rate change across a continuous state variable such as size. IPMs have multiple advantages including better performance with small datasets than traditional matrix models \cite{Ramula2009IntegralHerbs}, and direct incorporation of covariates of interest directly into vital rate models. We built these models with two objectives in mind.

Our first objective was to determine if including information about the seed bank significantly altered population models for \textit{O. coloradensis}. Seed banks can serve as important reservoirs of genetic diversity and buffer populations against collapse \cite{Vitalis2004WhenPerenniality, Jongejans2006WhatRange}, and can be especially critical for monocarpic perennials such as \textit{O. coloradensis} that only flower once in their lifetime \cite{Rees2006}. For these reasons, we expected that a soil seed bank is important for \textit{O. coloradensis}. Seed banks are often not included in population models because their parameters can be very difficult to estimate, but previous work shows that including them can significantly alter model outcomes \cite{Paniw2017, Nguyen2019ConsequencesModels}. We predicted that including a discrete seed bank state in IPMs would increase the $\lambda$ for \textit{O. coloradensis} populations. 

Our second objective was to identify whether any of the five aforementioned persistence mechanisms was acting to maintain \textit{O. coloradensis} populations. This species occurs in habitats that naturally experience frequent, highly localized disturbance, meaning that some subpopulations might be negatively affected by flood, for example, while other nearby subpopulations are simultaneously thriving due to lack of disturbance. Additionally, previous matrix population models constructed for this species in the 1990s found substantial variation in $\lambda$ across space and time \cite{Floyd1998}. The population-wide pattern of asynchronous habitat disturbance also could make source-sink dynamics important. Finally, we have evidence of large fluctuations in the number of plants within subpopulations \cite{Heidel202133-YearWyoming}, which suggest that population growth rate decreases at high population size and increases at low population size. Therefore, we predicted that density dependence, small-scale source-sink dynamics and asynchronous responses between subpopulations would be important mechanisms of persistence for \textit{O. coloradensis}.  

\begin{figure}[h!]
  \centering
  \includegraphics[width=1\textwidth]{COBP_conceptualDiagram.pdf}
  \caption{
  \internallinenumbers The evidence that would be required to show support for each of the five mechanisms that can contribute to the long-term viability of small populations of rare species.}
  \label{fig:conceptualFigure}
\end{figure} 

\section{Materials and Methods}

\subsection{Species Description}  
\textit{Oenothera coloradensis} (Onagraceae) \cite{Wagner2013-ii} is an herbaceous, monocarpic perennial plant species that primarily occurs in frequently disturbed, mesic or wet meadows, and riparian floodplains
\cite{Fertig2000-ow}. Non-reproductive individuals consist of a rosette of leaves with a fleshy taproot. Flowering typically occurs after several years, when individuals bolt and produce a 10-30 cm long floral stalk. Individuals typically die after reproducing--93\% of the time in populations we observed. Frequent disturbance such as flooding that reduces growth of both woody and herbaceous species and removes litter is important for this species, especially for successful seedling recruitment \cite{Fertig2000-ow, Burgess2003}. 

All historical and known extant \textit{O. coloradensis} populations lie within a 7,000-hectare area that includes southeast Wyoming, northern Colorado, and a small part of southwest Nebraska (Fig. S1). The only known population on Federal land occurs on the F. E. Warren Air Force Base near Cheyenne, WY (FEWAFB). The Soapstone Prairie Natural Area (Soapstone), owned by the city of Fort Collins, CO, has the largest known population of \textit{O. coloradensis} individuals \cite{Heidel202133-YearWyoming}. Decline in a majority of the known populations between the mid-1980s and 2000 lead the U.S. Fish and Wildlife Service (USFWS) to designate \textit{O. coloradensis} as a “threatened” species protected under the Endangered Species Act in 2000 (Endangered and Threatened Wildlife and Plants, 2000). Although this species appears to be naturally rare, mangers were concerned that habitat loss due to ranching, natural resource extraction, and shrub encroachment may lead to extinction of this species. However, based on additional monitoring following the initial listing decision, the USFWS determined that \textit{O. coloradensis} populations exhibit considerable natural variation in size, and that while monitored populations have both increased and decreased since the initial listing decision, the species as a whole does not appear to be on a trajectory toward extinction. As a result, \textit{O. coloradensis} was de-listed in 2019 (Endangered and Threatened Wildlife and Plants, 2019).
\nocite{USFWS2019}
\nocite{USFWS2000}

Previous work established that \textit{O. coloradensis} population growth rate is particularly impacted by recruitment of seedlings \cite{Floyd1998}. Seed banks are also likely important, since years of high seedling density are not necessarily preceded by years of high rates of flowering and seed production \cite{Munk2002RosetteSpecies, Heidel202133-YearWyoming}. The \textit{O. coloradensis} seed bank has not been studied directly, but a greenhouse seed study showed that an average of 58\% of seeds produced by a parent plant are viable, and that a viable seed has a 20\% probability of germinating after two months of cold stratification \cite{Burgess2005CapsuleColoradensis}. These rates did not change meaningfully over five years. More information about \textit{O. coloradensis} can be found in the Supplementary Material: "Species Information."

\subsection{Demographic Data Collection}
 We conducted a three-year demographic study of \textit{O. coloradensis} across six spatially distinct subpopulations, three in the FEWAFB population and three at the Soapstone population ("Unnamed creek", "Crow creek", and "Diamond creek" at FEWAFB and "Meadow", "HQ3" and "HQ5" at Soapstone)(Table S1; Fig. S1). In early summer 2018, we established three 2\textsf{x}2 m$^2$ quadrats in each of these subpopulations, resulting in 18 plots (Table S1). Plants larger than 3 cm are typically "non-seedling" plants at least one year in age. In each study plot, we tagged and mapped each unique "non-seedling" individual and recorded longest leaf length, reproductive status (bolting and/or flowering), and seed production for each. Individuals smaller than 3 cm in leaf length are typically seedlings that germinated that year, occur at high density, and are less likely to survive than non-seedling plants. Due to these factors, we tallied seedlings in each plot, but did not map or tag them. In subsequent 2019 and 2020 censuses, we mapped and tagged new "non-seedling" individuals, and re-measured all surviving individuals from previous years. Sample size in a given year at a subpopulation ranged from 48 to 1527 individuals (Table S1). All mapping, tagging, and leaf measurements took place between late May and early July, during the peak of vegetative growth for this species. We conducted a second round of site visits in early fall at the end of flowering, during which we confirmed that plants  designated as "reproductive" during the summer census did in flower, and collected data on seed production. 
 
It was not possible to measure seed production exactly because \textit{O. coloradensis} seeds are contained in indehiscent capsules. Additionally, buds on the same individual flower and set seed with a time lag of up to several weeks, so mature seed capsules often exist at the tip of a stem while un-opened buds lower down on that same stem have not yet flowered. This lag makes it difficult to count the total number of capsules produced by an individual. However, seed capsules leave a noticeable scar on the stem, so we used the number of seed capsule scars on reproductive stems as an estimate of capsule production. Counting scars is extremely time-intensive since a single plant can produce several hundred capsules, so we used Poisson generalized linear regression to estimate the relationship between the length of stem bearing capsule scars and the number of capsules produced by that stem. A Poisson regression model fit to stem measurements and capsule counts from 106 individuals in 2018 indicated that the number of capsules produced by an individual (C) can be predicted by $e^{(1.843 + 0.119\text{\textsf{x}}\text{S}))}$, where S is the stem length in cm (pseudo $R^2$ = 0.42, \textit{P} $<$ 0.01, Residual deviance = 186.98, df = 104) (Fig. S2). We used this relationship to estimate capsule production for each reproductive individual. Previous work indicated that each capsule contained an average of 4 seeds, so we multiplied the estimated number of capsules produced by an adult plant by 4 to estimate seed production \cite{Burgess2005CapsuleColoradensis}. 

\subsection{Environmental Measurements}
To determine the effect of temporal variation in climate on \textit{O. coloradensis} populations, we used modeled, population-level temperature and precipitation data from PRISM \cite{PRISMClimateGroupOregonStateUniversity2021PRISMUniversity}, which we refer to as "environmental covariates". We calculated the mean temperature of both the growing season and preceding winter season for each year of vital rate data collection at FEWAFB and Soapstone Prairie. We also calculated total precipitation for each "water year", the period from October of the previous year to September of the current year. We used this metric in place of growing season precipitation because the shortgrass steppe ecosystem in which \textit{O. coloradensis} occurs receives a majority of its annual precipitation in the form of snow, and melting snow from the previous winter likely drives springtime seedling recruitment. Average temperature of the previous winter is also likely important for seedling recruitment, because seed germination is triggered by cold stratification \cite{Burgess2005CapsuleColoradensis}. Growing season temperature and precipitation are likely important for growth, survival, and reproductive output of non-seedling plants. We also calculated the standard deviation of growing season and winter temperature and water year precipitation, which we used to quantify the variability of these variables over time. 

\subsection{Vital Rate Models}
We used data from the three-year demographic monitoring study detailed above to parameterize models of \textit{O. coloradensis} vital rates (shown in Fig. \ref{fig:lifecycle}; parameters of fitted vital rate functions are shown in Table S2). We first estimated continuous vital rate functions describing how survival probability, growth, flowering probability, and seed production in year \textit{t+1} each vary as a function of longest leaf size in year \textit{t}. We also initially included a quadratic term for longest leaf size in year \text{t} in these models, but AIC model selection determined that this term only improved models for flowering probability. We also estimated a continuous vital rate function describing the distribution of new recruit size in year \textit{t+1}. Finally, we estimated discrete vital rate parameters describing the probability of seeds produced in year \textit{t} either entering the seed bank or germinating in year \textit{t+1}, as well as the probability of seeds in the seed bank in year \textit{t} either staying in the seed bank or germinating in year \textit{t+1}.  

We first created "global" models for each continuous vital rate (Table. \ref{table:VitalRates}). 
We modeled survival probability ($s(z)$) as a function of log-transformed leaf $size_t$ using generalized linear models with binomial error distributions. Flowering individuals were excluded from the data used to fit survival models, since \textit{O. coloradensis} is a monocarpic perennial that nearly always dies after flowering.
We modeled probability of flowering ($Pb(z)$) using generalized linear models with binomial error distributions, and was predicted by log-transformed leaf $size_t$ plus log-transformed leaf $size_t$ squared. 
We modeled seed production ($b(z)$) as a function of $size_t$ using negative binomial models because the count data was over-dispersed. We only used data from flowering plants to fit these models, using the “glm.nb” function from the “MASS” R package \cite{Venables2002ModernS}.
We described growth, or the distribution of plant size in year ${t+1}$ ($G(z',z)$) as a series of Normal distributions with mean = $\mu_s$ and standard deviation = $\sigma_s$. $\mu_s$ was modeled as a function of $size_t$ using a linear model with Gaussian error. $\sigma_s$ was the residual standard error of this linear model. 
We described the distribution of recruit size in $year_{t+1}$ ($c_o(z')$) as a Normal distribution with the mean $\mu_r$, and the standard deviation $\sigma_r$. $\mu_r$ and $\sigma_r$ were the mean and standard deviation of observed longest leaf size in year ${t+1}$.

We then used these "global" model structures to fit different versions of these continuous vital rate functions, each of which described vital rate processes at different temporal and spatial scales. Models were fit using data from the first transition (2018-2019), the second transition (2019-2020), or pooled across both transitions. We also made models using data from a single subpopulation, a single population, or pooled across both populations. We additionally fit models that expanded on the "global" model structures by including density dependence terms and/or environmental covariates (total or standard deviation of water year precipitation, mean or standard deviation of annual growing season temperature, or mean or standard deviation of annual winter temperature). When density dependence or environmental covariates were included, we used AIC model selection to confirm that including these covariates improved model fit.   

All continuous vital rate models, regardless of scale, were parameterized using data from “non-seedling plants” as well as seedlings. Although seedlings (above-ground plants $<$ 3 cm in leaf length) were only tallied in each plot quadrant and year instead of tagged and measured, we incorporated them into the dataset for continuous, above-ground plants by assigning them a random size drawn from a continuous, uniform probability distribution (seedling size $\sim U($0.1, 3)). Each new recruit to the $>$ 3 cm stage in year $t+1$ was randomly assigned to a seedling within the same plot quadrant in year $t$. Seedlings in year $t$ that were assigned a recruit in year $t+1$ survived, while those without an assigned recruit died. Incorporating seedlings into the continuous dataset in this fashion allowed us to create IPMs using only one discrete stage.

We estimated discrete vital rates for seeds uniformly across both populations and years, using data from both greenhouse and field-based germination and seed viability studies. (Table \ref{table:VitalRates}). We did not have the data required to determine how these rates changed across subpopulations or in response to abiotic variation, due to the difficulties of estimating \textit{in situ} seed germination and death. We used the following parameters to estimate discrete seed vital rate parameters: viable seed germination rate (germ. rate) = 0.16, viability rate of seeds produced by a parent plant (viab. rate) = 0.58, rate of natural seed death in the seed bank (death rate) = 0.10. More information about discrete vital rate parameter estimation can be found in "Supplementary Information: Discrete Vital Rate Parameter Estimation." 

\begin{figure}[ht!]
    \centering
    \includegraphics[width=1\textwidth]{figures/COBP_lifecyclediagram_new.pdf}
    \caption{\internallinenumbers
    Diagram of the \textit{O. coloradensis} life-cycle, with transitions labeled with the notation used in vital equations in Table \ref{table:VitalRates}. Based on model structures and notation from: (Paniw et al., 2017; Merow et al., 2014; Ellner et al., 2016). "germ.rt" = germination rate, "viab.rt" = viability rate, "death.rt" = death rate.}
 \label{fig:lifecycle}
\end{figure}

\begin{table}[h!]
\centering
\begin{spacing}{1.3}
\caption{
Description of vital rates used in \textit{O. coloradensis} IPMs\label{table:VitalRates}}
\begin{tabular}{L{0.16\textwidth}p{0.28\textwidth}p{0.45\textwidth}}
\toprule
Vital Rate & Description & Model \\
\hline
\rowcolor[gray]{.95} \textit{pEstab} & \small\textit{P}(seed produced in \textit{t} establishes as a seedling in \textit{t+1}) & $pEstab$ = $\frac{Num.\;new\;recruits\;in\;year_{t+1}}{Num.\;seeds\;produced\;in\;year_t}$\\
\textit{goCont} & \small \textit{P}(seed produced in \textit{t} germinates in \textit{t+1}) & $goCont$ = viab. rate \textsf{x} germ. rate\\
\rowcolor[gray]{.95} 
\textit{outSB} & \small\textit{P}(seed bank seed in \textit{t} germinates in \textit{t+1}) & $outSB$ = germ. rate \textsf{x} ($1 - $death rate)\\
\textit{goSB} & \small\textit{P}(seed produced in \textit{t} goes into the seed bank in \textit{t+1}) & $goSB$ = viab. rate \textsf{x} ($1 - $germ. rate)\\
\rowcolor[gray]{.95} 
\textit{staySB} & \small\textit{P}(seed bank seed in \textit{t} stays in the seed bank in \textit{t+1}) & $staySB$ = ($1-$ germ. rate) \textsf{x} ($1 - $death rate)\\
\textit{Survival (s(z))} & \small\textit{P}(survival from \textit{t} to \textit{t+1}) & logit(survival) $\sim \beta_0 + \beta_1 (ln(size_t)) + \epsilon$\\ 
\rowcolor[gray]{.95} 
 \textit{Flowering (Pb(z))} & \small\textit{P}(flowering in \textit{t}) & logit(flowering) $\sim \beta_0 + \beta_1 (ln(size_t)) + \beta_2 (ln(size_t)^2) + \epsilon$\\ 
 \textit{Seed prod. (b(z))} & \small Seed production in \textit{t} & exp(seed number) $\sim \beta_0 + \beta_1 (ln(size_t)) +  \epsilon$\\ 
\rowcolor[gray]{.95} 
\textit{Growth (G(z',z))} & \small Distribution of longest leaf size in year \textit{t} & $G(z',z) = N(\mu_s, \sigma_s); \newline \mu_s \sim \beta_0 + \beta_1 (ln(size_t)) + \epsilon; \newline \sigma_s \sim RSE(\beta_0 + \beta_1 (ln(size_t)) + \epsilon)$\\ 
 \textit{Recruit size ($c_o$(z'))} & \small Distribution of new recruit size in year \textit{t} & $c_o(z') = N(\mu_r, \sigma_r); \newline \mu_r = $mean(size of recruits in $t); \newline \sigma_r = $stnd. dev. (size of recruits in $t)$\\
\hline
\multicolumn{3}{l}{* RSE = residual standard error} 
\end{tabular}
\end{spacing}
\end{table}

\subsection{Population Models}
We used estimates of discrete and continuous \textit{O. coloradensis} vital rates, detailed above, to parameterize a suite of integral projection models (IPMs) for \textit{O. coloradensis}. We then used these models to address each of the objectives outlined in the Introduction.

\textit{Objective 1: Quantifying the Importance of the Seed bank Stage}: We used two different IPMs to determine whether explicitly including a discrete seed bank stage in a population model leads to significantly different outcomes relative to a model without a seed bank stage. We first created a density-independent IPM using continuous vital rate functions parameterized with data from both Soapstone and FEWAFB. This model had a single continuous, size-based population state, and did not include a seed bank state (Table \ref{IPMsTable}: IPM “A”; Eqn. \ref{eqn:oneStateIPM}). This IPM used a kernel structure where the continuous, above-ground population state ($n(z', t+1)$) at time $t+1$ was described by the following equation:  
\begin{equation}\label{eqn:oneStateIPM} 
n(z', t+1) = \int_{L}^{U}(1-P_b(z))s(z)G(z',z)n(z,t)dz + pEstab\int_{L}^{U}P_b(z)b(z)c_o(z')n(z,t)dz
\end{equation}

Then, we created an IPM that included both a discrete seed bank state, and a continuous, size-based stage for above-ground individuals (Table \ref{IPMsTable}: IPM “B”; Eqns. \ref{eqn:twoStateIPM_cont} \& \ref{eqn:twoStateIPM_disc}) \cite{Ellner2006IntegralDemography, Rees2006, Paniw2017}. This model used the same continuous vital rate functions as in IPM A, but also included discrete probabilities describing the probabilities of seeds produced in year \textit{t} germinating or going into the seeds d bank in year \textit{t+1}, as well as probabilities of seeds in the seed bank in year \textit{t} germinating or persisting in the seed bank in year \textit{t+1}. This IPM with two population states used a kernel structure where the continuous, above-ground population state ($n(z',t+1)$) and the seed bank state ($B(t+1)$) at time $t+1$ were described by the following equations: 

\begin{equation}
\label{eqn:twoStateIPM_cont}
\begin{split}
n(z', t+1) = \int_{L}^{U}(1-P_b(z))s(z)G(z',z)n(z,t)dz + \\ goCont\int_{L}^{U}P_b(z)b(z)c_o(z')n(z,t)dz + outSB 
\end{split}
\end{equation}

\begin{equation}\label{eqn:twoStateIPM_disc}
     B(t+1) = goSB\int_{L}^{U}P_b(z)b(z)n(z,t)dz + B(t)staySB 
\end{equation}

%\begin{table}
\small

\begin{spacing}{1.2}
\centering
%\resizebox{\textwidth}{!}{ 
\begin{longtable}[ht!]
{|p{0.028\textwidth}|p{0.02\textwidth} p{0.02\textwidth}|p{0.02\textwidth}| p{0.018\textwidth} p{0.018\textwidth} | p{0.018\textwidth} p{0.018\textwidth} p{0.018\textwidth} p{0.018\textwidth} p{0.018\textwidth} p{0.018\textwidth} 
|p{0.02\textwidth}|p{0.02\textwidth} p{0.02\textwidth}|p{0.04\textwidth}|p{0.04\textwidth}|p{2.55cm}|}   
\caption{ A description of the data used to create each IPM, as well as the covariates included in the vital rate models used in that IPM. ln($\lambda$) estimates and 95\% bootstrap confidence intervals of ln($\lambda$) are also shown for each IPM.\label{IPMsTable}} \\
%\hline \\[-1.6ex] 
\toprule
\rotatebox{90}{IPM} & 
\multicolumn{11}{c}{Data Included} & 
\multicolumn{3}{|c}{Transition} & 
\multicolumn{2}{|c|}{Covariates} & 
\small ln($\lambda$) \hspace{2em} \footnotesize (95\% CI) 
\\ \cline{2-18}
& & & & \multicolumn{2}{|p{0.05\textwidth}|}{Each pop.} & \multicolumn{6}{c|}{Each subpop.} & & & & & &  \\\cline{5-12}
& \rotatebox{90}{Continuous state only} & \rotatebox{90}{Continuous + seed bank state} & \rotatebox{90}{All subpopulations} & \rotatebox{90}{Soapstone} & \rotatebox{90}{FEWAFB} & \rotatebox{90}{Unnamed Creek} & \rotatebox{90}{Diamond Creek} & \rotatebox{90}{Crow Creek} & \rotatebox{90}{Meadow} & \rotatebox{90}{HQ3} & \rotatebox{90}{HQ5} & \rotatebox{90}{All Transitions} & \rotatebox{90}{2018-2019} & \rotatebox{90}{2019-2020} & \rotatebox{90}{Density dependence} & \rotatebox{90}{Environmental covariates} &  \\
\hline
\rowcolor[gray]{.95} A&\textsf{x}&&\textsf{x}&&&&&&&&&\textsf{x}&&&&&0.27 \hspace{2em} \footnotesize (0.269, 0.271)  \\
B &&\textsf{x}&\textsf{x}&&&&&&&&&\textsf{x}&&&&&0.65 \hspace{2em} \footnotesize (0.648, 0.650)   \\
\rowcolor[gray]{.95}C&&\textsf{x}&&&&\textsf{x}&&&&&&\textsf{x}&&&&&0.48 \hspace{2em} \footnotesize  (0.477, 0.489) \\
D &&\textsf{x}&&&&&\textsf{x}&&&&&\textsf{x}&&&&& 1.13 \hspace{2em} \footnotesize  (1.124, 1.142)  \\
\rowcolor[gray]{.95} E &&\textsf{x}&&&&&&\textsf{x}&&&&\textsf{x}&&&&&0.74 \hspace{2em} \footnotesize  (0.725, 0.746) \\
F &&\textsf{x}&&&&&&&\textsf{x}&&&\textsf{x}&&&&&0.54 \hspace{2em} \footnotesize  (0.520, 0.551)  \\
\rowcolor[gray]{.95} G &&\textsf{x}&&&&&&&&\textsf{x}&&\textsf{x}&&&&&0.395 \hspace{2em} \footnotesize (0.378, 0.401)  \\
H &&\textsf{x}&&&&&&&&&\textsf{x}&\textsf{x}&&&&&0.53 \hspace{2em} \footnotesize (0.526, 0.540)  \\
\rowcolor[gray]{.95} I &&\textsf{x}&&&&\textsf{x}&&&&&&\textsf{x}&&&\textsf{x}&&0.59$^\dag$ \hspace{2em} \footnotesize (0.576, 0.637)  \\
J &&\textsf{x}&&&&&\textsf{x}&&&&&\textsf{x}&&&\textsf{x}&&0.63$^\dag$  \hspace{2em} \footnotesize (0.611, 0.723)  \\
\rowcolor[gray]{.95} K &&\textsf{x}&&&&&&\textsf{x}&&&&\textsf{x}&&&\textsf{x}&& $-0.10$$^\dag$ \hspace{2em} \footnotesize ($-0.135$, 0.063)  \\
L &&\textsf{x}&&&&&&&\textsf{x}&&&\textsf{x}&&&\textsf{x}&&$-0.20$$^\dag$  \hspace{2em} \footnotesize ($-0.229$, $-0.167$)  \\
\rowcolor[gray]{.95} M &&\textsf{x}&&&&&&&&\textsf{x}&&\textsf{x}&&&\textsf{x}&&1.31$^\dag$  \hspace{2em} \footnotesize (1.294, 1.354)  \\
N &&\textsf{x}&&&&&&&&&\textsf{x}&\textsf{x}&&&\textsf{x}&&2.31$^\dag$ \hspace{2em} \footnotesize (2.297, 2.33)  \\
\rowcolor[gray]{.95} S &&\textsf{x}&&&&\textsf{x}&&&&&&\textsf{x}&&&\textsf{x}&\textsf{x}&0.58$^\dag$ \\
T &&\textsf{x}&&&&&\textsf{x}&&&&&\textsf{x}&&&\textsf{x}&\textsf{x}&0.51$^\dag$ \\
\rowcolor[gray]{.95} U &&\textsf{x}&&&&&&\textsf{x}&&&&\textsf{x}&&&\textsf{x}&\textsf{x}&0.90$^\dag$ \\
V &&\textsf{x}&&&&&&&\textsf{x}&&&\textsf{x}&&&\textsf{x}&\textsf{x}&$-0.27$$^\dag$ \\
\rowcolor[gray]{.95} W &&\textsf{x}&&&&&&&&\textsf{x}&&\textsf{x}&&&\textsf{x}&\textsf{x}&$-0.18$$^\dag$ \\
X &&\textsf{x}&&&&&&&&&\textsf{x}&\textsf{x}&&&\textsf{x}&\textsf{x}&0.76$^\dag$ \\
\rowcolor[gray]{.95} AA &&\textsf{x}&&\textsf{x}&&&&&&&&\textsf{x}&&&&&0.50 \hspace{2em} \footnotesize (0.497, 0.501)  \\
BB &&\textsf{x}&&&\textsf{x}&&&&&&&\textsf{x}&&&&&0.73 \hspace{2em}  \footnotesize (0.729, 0.733)  \\
\rowcolor[gray]{.95} CC &&\textsf{x}&&&&\textsf{x}&&&&&&&\textsf{x}&&&&0.38 \hspace{2em} \footnotesize (0.370, 0.388)  \\
DD &&\textsf{x}&&&&&\textsf{x}&&&&&&\textsf{x}&&&&1.56 \hspace{2em} \footnotesize (1.545, 1.572)   \\
\rowcolor[gray]{.95} EE &&\textsf{x}&&&&&&\textsf{x}&&&&&\textsf{x}&&&&0.90 \hspace{2em} \footnotesize (0.864, 0.904)  \\
FF &&\textsf{x}&&&&&&&\textsf{x}&&&&\textsf{x}&&&&0.62 \hspace{2em} \footnotesize (0.592, 0.637)   \\
\rowcolor[gray]{.95} GG &&\textsf{x}&&&&&&&&\textsf{x}&&&\textsf{x}&&&&0.73 \hspace{2em} \footnotesize (0.727, 0.753)  \\
HH &&\textsf{x}&&&&&&&&&\textsf{x}&&\textsf{x}&&&&1.11 \hspace{2em} \footnotesize (1.108, 1.126)  \\
\rowcolor[gray]{.95} II &&\textsf{x}&&&&\textsf{x}&&&&&&&&\textsf{x}&&&0.50 \hspace{2em} \footnotesize (0.492, 0.513)  \\
JJ &&\textsf{x}&&&&&\textsf{x}&&&&&&&\textsf{x}&&&0.71 \hspace{2em} \footnotesize (0.692, 0.726)   \\
\rowcolor[gray]{.95} KK &&\textsf{x}&&&&&&\textsf{x}&&&&&&\textsf{x}&&&0.76 \hspace{2em} \footnotesize (0.739, 0.774)   \\
LL &&\textsf{x}&&&&&&&\textsf{x}&&&&&\textsf{x}&&&0.41 \hspace{2em} \footnotesize (0.378, 0.448)   \\
\rowcolor[gray]{.95} MM &&\textsf{x}&&&&&&&&\textsf{x}&&&&\textsf{x}&&&0.03   \hspace{2em} \footnotesize (0.013, 0.040)   \\
NN &&\textsf{x}&&&&&&&&&\textsf{x}&&&\textsf{x}&&&$-0.10$ \hspace{2em} \footnotesize ($-0.112$, $-0.097$)    \\
\hline
\multicolumn{18}{p{.96\textwidth}}{*Note: We did not calculate bootstrap 95\% confidence intervals for ln($\lambda$) of models “S” –“X”, since only vital rate parameters and not lambda values from these models were used in further analysis. }\\
\multicolumn{18}{p{.96\textwidth}}{\dag: These values show stochastic lambda ($ln(\lambda_s)$). Other values are asymptotic lambda ($ln(\lambda_a)$).}
\end{longtable}
\end{spacing}

In equations for both types of IPMs, $z$ is the distribution of plant longest leaf size (measured as longest leaf length) in the current year (“$size_t$”), $z’$ is the distribution of plant longest leaf size in the next year (“$size_{t+1}$”), and $U$ and $L$ are the upper and lower boundaries of leaf size. $G(z’, z)$ is the vital rate function describing $size_{t+1}$ as a function of $size_t$. The vital rate functions $s(z)$, $Pb(z)$, and $b(z)$ describe the relationship between $size_t$ and survival probability of non-flowering plants, flowering probability, and seed production of flowering plants, respectively.  $c_o(z’)$ is the distribution of above-ground recruit $size_{t+1}$. $goCont$, $outSB$, $goSB$, and $staySB$ are discrete parameters that determine seed bank dynamics. $goCont$ is the probability of a seed produced in year $t$ germinating as a seedling in year $t+1$, $outSB$ is the probability of a seed from the seed bank in year $t$ germinating as a seedling in year $t+1$, $goSB$ is the probability of a seed produced in year $t$ going into the seed bank in year $t+1$, and $staySB$ is the probability of a seed from the seed bank in year $t$ persisting in the seed bank in year $t+1$ \cite{Paniw2017} (Table \ref{table:VitalRates}). $pEstab$ is the probability of a seed produced in year $t$ establishing as a seedling in year $t+1$, and is only used in the IPM with no seedbank stage.  

We used these vital rate functions and discrete parameters described above to construct discretized IPM kernels for IPM "A" and IPM "B". All kernels were numerically implemented using the “midpoint rule” method \cite{Easterling2000} with 500 bins, an upper size limit corresponding to 120\% of the maximum observed longest leaf size and a lower size limit corresponding to 80\% of the minimum simulated seedling size of 0.1 cm. We corrected for eviction following methods from \cite{Williams2012-xs}. We then used eigen analysis of these kernels to estimate asymptotic population growth rate ($\lambda$), damping ratio, stable size distribution, and reproductive value \cite{Caswell2001MatrixInterpretation, Ellner2016Data-drivenPopulations}. We used 1000 iterations of nonparametric bootstrap re-sampling to estimate 95\% bootstrap confidence intervals (95\% CIs) for each continuous vital rate parameter included in each IPM, as well as each estimate of $\lambda$ \cite{Merow2014AdvancingGuide, Fieberg2020Resampling-basedBiologists}. We were unable to estimate CIs for discrete seed bank parameters because they were drawn from a previous publication. We used perturbation analysis to determine the sensitivity and elasticity of $\lambda$ to changes in germination rate, viability rate, seed survival rate, and each continuous vital rate model \cite{Morris2002QuantitativeAnalysis}. Finally, to determine determine whether including a discrete seed bank state significantly altered our population model, we compared the asymptotic $\lambda$ and associated 95\% CI between IPM "A" and IPM "B." 

\subsection{Objective 2: Evaluating Persistence Mechanisms}
To evaluate whether any of the demographic mechanisms of rare species persistence outlined in Fig. \ref{fig:conceptualFigure} are acting in populations of \textit{O. coloradensis}, we fit a series of IPMs that each used different subsets of data and/or additional covariates in vital rate functions (Table \ref{IPMsTable}: IPMs “C” – “NN”). These IPMs all had a mathematical form equivalent to that of IPM "B" described above, with a discrete seed bank state, and a continuous, size-based stage for above-ground individuals (Eqns. \ref{eqn:twoStateIPM_cont} \& \ref{eqn:twoStateIPM_disc}). We then used each of these IPMs, as well as the vital rate functions used to construct them, to evaluate a different persistence mechanism. Details of this process for each persistence mechanism are provided below. It is important to note that, although we use "$\lamda$" to refer to population growth rate throughout the text, this value was calculated in slightly different ways depending on the type of IPM. For IPMs without density dependence or environmental covariates, we calculated asymptotic growth rate (ln($\lambda_a$)) using eigen analysis of the transition matrix (as described above for IPMs "A" and "B"). For IPMs that used vital rate models with coefficients for density dependence or environmental covariates, we used the ipmr R package to determine stochastic growth rate (ln($\lambda_s$)) through iteration (\cite{Levin2021Ipmr:R}). 

\textit{Negative Density Dependence:} In order to determine the importance of density dependence in \textit{O. coloradensis} subpopulations, we used IPMs and vital rate functions that were fit uniquely for each subpopulation using data from both transitions. However, one set of IPMs included population size in the current year in vital rate models, while another set of IPMs did not (density-independent IPMs: “C”-“H” in Table \ref{IPMsTable}; density-dependent IPMs: “I”-“N”). We used AIC to identify significant differences between vital rate models with and without density dependence terms. We also used results from subpopulation-level IPMs that did not include covariates for density dependence (Table \ref{IPMsTable}: IPMs "CC"-"NN") for each transition to identify relationships between subpopulation size in year \textit{t}  and ln($\lambda$) (as in Fig. \ref{fig:conceptualFigure}), as well as subpopulation size in year \textit{t} and the ratio of population size in year \textit{t+1 }and subpopulation size in year \textit{t}. In addition to population size information and ln($\lambda$) values from our IPMs, we also used population sizes and ln($\lambda$) values from a previously-published demographic study of \textit{O. coloradensis} at the three FEWAFB subpopulations that we also monitored \cite{Floyd1998}.  A negative relationship between population size in year \textit{t} and either ln($\lambda$) or the ratio of population size in year \textit{t + 1} to population size in year \textit{t} would provide evidence for negative density dependence. Additionally, significant differences between models with and without population size predictor terms would constitute evidence for density dependence.  

\textit{Demographic Compensation:} To test for demographic compensation, we calculated the correlation between environmental covariate coefficients in different vital rate models. For this correlation analysis we used vital rate models that were fit using data from each subpopulation and both transitions, and that included covariates for density dependence and as well as environmental covariates that improved model fit (vital rate models from IPMs “S”-“X” in Table \ref{IPMsTable}). A negative correlation between coefficients of the same covariate in different vital rate models would indicate that demographic compensation was taking place \cite{Villellas2015DemographicImplications, Dibner2019}. For example, if soil moisture had a positive effect on growth but a negative effect on survival, this would be evidence for demographic compensation. We tested the significance of negative correlations between environmental covariate coefficients using a randomization procedure similar to that used by Villellas et al. (2015), where we randomly assigned an environmental covariate coefficient drawn from the observed distribution of values for that coefficient to each vital rate function, calculated a correlation matrix between those coefficients in each vital rate function, and counted the number of negative correlations in that matrix. This procedure was repeated 10,000 times to generate a null distribution of the expected number of negative correlations between environmental coefficients that would occur randomly. We compared the observed number of negative correlations between each environmental covariate coefficient to these expected distributions of random correlations to determine statistical significance. We could not test for demographic compensation in discrete seed bank vital rate parameters because we did not know how they varied according to environmental conditions.  A significant negative correlation between environmental covariate coefficients in different vital rate models would provide evidence for demographic compensation (Fig. \ref{fig:conceptualFigure}).  


\textit{Vital Rate Buffering:} We tested for the presence of vital rate buffering in \textit{O. coloradensis} populations by comparing the variability of vital rates to their importance. We used an approach that scales both the standard deviation (variability metric) and sensitivity (importance metric) of vital rates, allowing for a fair comparison of variability and importance across vital rates with fundamentally different relationships between their mean and variance \cite{McDonald2017DivergentEnvironments}. Vital rates that are probabilities (i.e. survival, flowering, growth, discrete seed bank transition probabilities, and seedling size) are constrained between zero and one and thus typically have small variance as the mean approaches these limits, while other vital rates are only constrained by zero and thus typically have variances that increase as the mean increases (i.e. seed productivity) \cite{Gaillard2003-ch}. To enable a fair comparison between these different categories of vital rates, we calculated the importance and variability of probability and non-probability vital rates in different ways. The importance of probability vital rates was defined as the logit variance stabilized sensitivity, and the variability was defined by the standard deviation of the logit transformed vital rate values \cite{McDonald2017DivergentEnvironments, William_A_Link_Paul_F_Doherty_Fr2002-cb}. The importance of non-probability vital rates was defined as the log-scaled sensitivity (or elasticity), and the variability was defined by the standard deviation of the log-transformed vital rate values \cite{McDonald2017DivergentEnvironments, Morris2002QuantitativeAnalysis}.

We used an IPM that was fit across all subpopulations using data from both transitions (Table \ref{IPMsTable}: IPM “B”) to calculate elasticity or logit VSS values for each discrete vital rate and continuous vital rate function. 
We calculated the scaled standard deviation for each continuous vital rate function using the vital rates that were fit uniquely for each subpopulation and each transition (Table \ref{IPMsTable}: IPMs “CC”-“NN”). Because we did not have site-level information about discrete seed bank vital rates, we simulated both the maximum and minimum possible standard deviations for each discrete vital rate. We then proceeded with two comparisons of vital rate variability and importance, once using the maximum possible discrete vital rate standard deviation, and another using the minimum. In order to determine the correlation between a single importance/variability value pair for discrete vital rates and a string of value pairs for continuous vital rate functions, we calculated mean importance and variability values for each continuous vital rate function. A significant negative correlation between the mean or absolute scaled importance (logit VSS or elasticity) and mean or absolute variability (standard deviation of logit or log-transformed vital rates) across all vital rates would constitute support for the presence of vital rate buffering in this species
(Fig. \ref{fig:conceptualFigure}).  

\textit{Asynchronous Responses and Source-Sink Dynamics:} To determine whether \textit{O. coloradensis} subpopulations showed asynchronous responses to environmental variation, we made a correlation matrix to determine how change in ln($\lambda$) across each transition was correlated across each subpopulation, using values of ln($\lambda$) derived from IPMs for each subpopulation (Table \ref{IPMsTable}: IPMs “C”-“H”). We used the “mantel()” function from the “vegan” R package to perform a Mantel test, which determined if the Spearman correlation of ln($\lambda$) across subpopulations was significantly related to the Euclidean distance between each subpopulation \cite{Oksanen2020Vegan:Package}. A negative relationship between the distance between subpopulations and degree of correlation of ln($\lambda$) would constitute evidence for spatial asynchrony between subpopulations (Fig. \ref{fig:conceptualFigure}).   

Because we did not have information about gene flow between subpopulations of \textit{O. coloradensis} via pollination or seed dispersal, it was not possible to directly measure whether fine-scale source-sink dynamics were acting in these populations. However, because variation in population growth rate across space is a prerequisite for source-sink dynamics, the previously described tests for spatial asynchrony in subpopulations can also provide evidence for the existence of source-sink dynamics. Again, this would be a negative relationship of distance between subpopulations and correlation of subpopulation ln($\lambda$) (Fig. \ref{fig:conceptualFigure}).  
 
\section{Results}

\subsection{Vital Rate Models} 

Larger non-reproductive plants were more likely to survive than smaller plants (Fig. \ref{fig:vitalRates}A). Plants below $\sim7.5$ cm were likely to be larger, while plants larger than $\sim7.5$ cm were likely to be smaller the following year (Fig. \ref{fig:vitalRates} B). Flowering probability was best approximated as a quadratic polynomial, where flowering probability peaked at 12 cm leaf length, and plants with the largest leaves exhibited low flowering probability (Fig. \ref{fig:vitalRates} C). The number of seeds that a reproductive plant produced increased sharply with leaf size (Fig. \ref{fig:vitalRates} D). The inclusion of additional covariates did not alter the overall shape or sign of the relationships between leaf size and vital rates, so models shown in Figure \ref{fig:vitalRates} did not include any additional covariates beyond leaf size.  

\subsection{Objective 1: Quantifying the Importance of the Seed bank Stage}

\textit{Integral Projection Models:} We found that including a discrete seed bank stage in IPMs for \textit{O. coloradensis} significantly increased the asymptotic population growth rate. The continuous state-only IPM (Table \ref{IPMsTable}: IPM “A”) predicted an asymptotic ln($\lambda$) of 0.27 for all populations (95\% CI: 0.269 - 0.271), while the continuous + discrete state IPM (Table \ref{IPMsTable}: IPM “B”) predicted an asymptotic ln($\lambda$) of 0.65 (populations (95\% CI: 0.648 - 0.650). All subsequent IPM results refer to models that included a discrete seed bank state. 

The simplest two-state IPMs that excluded density dependence and environmental variation indicated that both the Soapstone prairie and FEWAFB populations had positive population growth rates (Table \ref{IPMsTable}: Soapstone prairie- IPM “AA”, ln($\lambda$) = 0.50;  FEWAFB – IPM “BB”, ln($\lambda$) =  0.73). The Diamond Creek subpopulation at FEWAFB had the highest population growth rate from 2018 to 2020 (Table \ref{IPMsTable}: IPM “D”, ln($\lambda$) = 1.13), while the HQ3 subpopulation at Soapstone prairie had the lowest growth rate (Table \ref{IPMsTable}: IPM “G”, ln($\lambda$) = 0.395). We parameterized multiple other sets of IPMs that used different combinations of covariates in their vital rate models, and almost all identified a positive population growth rate (Table \ref{IPMsTable}).   


\begin{figure}[h!]
  \centering
  \includegraphics[width=\textwidth]{figures/vitalRateModelFit_withBoxplots.pdf}
  \caption{\internallinenumbers The effect of current year leaf size size$_t$ on vital rates in monitored \textit{O. coloradensis} populations. Data for from all sites and all transitions is shown. Lines indicate vital rate functions for each population, and include only size$_t$ as a predictor, with the exception of flowering models, which include a (size$_t$$^2$ term. Bands around each line show 95\% confidence intervals. In plots \textbf{A & B}, boxplots above the main panels indicate the distribution of leaf size for individuals that did not survive or flower (dark grey) or did survive or flower (light grey). Grey points indicate data for an individual plant in a given transition.  The dashed line in panel \textbf{B} shows a 1:1 line. The sharp cut-off in size$_{t+1}$ in panel \textbf{B} is due to the fact that two-year-old plants could not be seedlings, which were classified as any plant less than 3 cm in size. Note that while leaf size in cm is shown in these plots for ease of interpretation, these values have been back-transformed from the ln(leaf size) values that were used in models.}
  \label{fig:vitalRates}
\end{figure} 


A density-independent, discretized IPM kernel (made using IPM “B” in Table \ref{IPMsTable}) showed transition probabilities within and between the discrete and continuous stages of the \textit{O. coloradensis} life cycle when all populations and transitions were considered together (Fig. \ref{fig:IPMKernel} A). Relative to the rest of the kernel, there was a very high probability that seeds stay in the seed bank, as well as a large contribution of seeds from medium-sized adult plants to the seed bank in the next year. The rates at which seeds are produced by adult plants and stay in the seed bank had the most impact on population growth rate (Fig. \ref{fig:IPMKernel} C).

\begin{figure}[h!]
  \centering
  \includegraphics[width=\textwidth]{figures/KernelMultipanelFigure.pdf}
  \caption{\internallinenumbers Visualizations of the \textit{O. coloradensis} IPM kernels. (\textbf{A}) The IPM kernel for \textit{O. coloradensis}. This kernel shows a density-independent IPM constructed using all data from all transitions (IPM "B"). (\textbf{B}) Sensitivity of the IPM kernel. (\textbf{C}) Elasticity of the IPM kernel. In all panels, color indicates probability, with darker colors corresponding to higher probability, and lighter colors corresponding to lower probability. The dashed line shows a 1:1 line. Contour lines in all panels indicate the locations of highest values in each plot, and are labeled with the corresponding values (Panel \textbf{A} = probability, panel \textbf{B} = sensitivity, panel \textbf{C} = elasticity).}
  \label{fig:IPMKernel}
\end{figure} 
\subsection{Objective 2: Evaluating Persistence Mechanisms}

\textit{Negative Density Dependence:} We found evidence that negative density-dependence was occurring in subpopulations of \textit{O. coloradensis}. AIC comparison of continuous vital rate models indicate that density-dependent models are better predictors of the majority of vital rates than density-independent models in most subpopulations (Table \ref{Table:DDModResults}).  Models that included population size in the previous year as a covariate were better predictors of growth in five of six subpopulations. Density dependent models were better predictors of survival and seed production than density independent models in four out of six subpopulations, and density dependent models of flowering were better in one subpopulation. Recruit size distribution was not affected by density dependence—AIC model comparison did not indicate substantial differences, either negative or positive, between recruit size models with and without density dependence terms in any subpopulation. The vital rate models for the Meadow population at Soapstone Prairie were least affected by density dependence. Although density dependence is important for \textit{O. coloradensis} in many situations, it appears only to be acting to decrease lambda at high density (as in the highly dense Diamond Creek or HQ5 subpopulations), but not clearly increasing lambda at low density (as in the sparsely populated Meadow subpopulation). We also found that, within a subpopulation, population growth rate (ln($\lambda$)) was generally higher when subpopulation size was smaller(Fig. \ref{fig:DDFig} A). Similarly, there was a negative relationship within each subpopulation between subpopulation size in year \textit{t} and the ratio of subpopulation size in \textit{t+1} to subpopulation size in \textit{t} (Fig. \ref{fig:DDFig} B and D); Interestingly, these negative relationships are generally pronounced at the subpopulation level, but are weak when examining data across all subpopulations

\textit{Demographic Compensation:} Our analyses did not identify signatures of demographic compensation in \textit{O. coloradensis} populations. While there were negative correlations between the effect of mean growing season temperature on vital rates for five combinations of vital rates, none of these correlations were significant (Table \ref{Table:DemoComp}). The only significant correlation was positive. Ten thousand correlations of randomly assigned coefficients found that the number of negative correlations in a matrix can be described by a normal distribution with a mean of 4.97 and a standard deviation of 1.60. Using this distribution as a null model, there was a 50.7\% probability of observing five negative correlations. Although there is no significant evidence for demographic compensation, it is notable that the effect of mean growing season temperature on distribution of recruit size was negatively correlated with the effect of growing season temperature on all other vital rates. We were only able to compare coefficients across vital rate models for mean growing season temperature, because including precipitation and mean winter temperature as covariates resulted in overfitting in some cases.  

\begin{table}[h!]
\centering
\begin{spacing}{1.2}
\caption{ \internallinenumbers Comparison of vital rate models that do and do not include density dependence. The “DI” and “DD” rows contain AIC values for each vital rate model in each subpopulation for models that are density-independent (DI) and density-dependent (DD). The difference between the AIC of DI and DD models is shown in the $\Delta$AIC column. Bold text indicates that the $|\Delta|$AIC value is $>$ 3, which means that including a term for density dependence substantially changed that vital rate model. A positive $|\Delta|$AIC indicates that including density dependence improved the model, while a negative value indicates that including density dependence made model fit worse.\label{Table:DDModResults}}
\begin{tabular}{l l p{.08\textwidth} p{.1\textwidth} p{.1\textwidth}ccc}
\toprule
\multicolumn{2}{c}{Vital Rate Model} & \multicolumn{6}{c}{Subpopulation} 
\\ \cline{3-8}
& & Crow Creek & Diamond Creek & Unnamed Creek & HQ5 & HQ3 & Meadow \\ 
\hline
\rowcolor[gray]{.95} Survival & DI &     776.58 &  1012.68 & 2684.34 & 3242.63 & 716.66 & 166.13 \\
\rowcolor[gray]{.95} & DD &              757.84 &  905.39  & 26848.74 & 2922.91 & 637.84 & 166.83 \\ 
\rowcolor[gray]{.95} & $\Delta$AIC &     \textbf{18.74} &   \textbf{107.28} &  -0.41 &   \textbf{320.33} &  \textbf{78.82} &   -0.70 \\ 
Growth & DI &                            510.34 &  953.29 &  1098.95 & 1570.93 & 300.18 & 116.54 \\
& DD &                                   506.61 &  931.15 &  1068.14 & 1112.78 & 269.73 & 113.88 \\
& $\Delta$AIC &                          \textbf{3.73} &    \textbf{22.15} &   \textbf{30.811} &  \textbf{458.15} &  \textbf{30.45} &  2.66 \\
\rowcolor[gray]{.95} Flowering & DI &    371.68 &  523.30 &  1087.93 & 538.52 &  191.46 & 104.24 \\
\rowcolor[gray]{.95} & DD &              373.31 &  523.74  & 1087.48 & 483.99 &  193.22 & 106.96 \\
\rowcolor[gray]{.95} & $\Delta$AIC &     -1.63 &   -0.44 &   0.45 &    \textbf{54.52} &   -1.76 &  -1.72 \\
Seed production & DI &                   842.00 &  1580.85 & 2815.89 & 1423.02 & 598.75 & 280.09 \\
& DD &                                   835.59 &  1566.83 & 2817.19 & 1419.32 & 594.63 & 281.45 \\
 & $\Delta$AIC &                         \textbf{6.41} &    \textbf{14.02} &   -1.29 &   \textbf{3.71} &    \textbf{4.12} &   -1.35 \\ 
\rowcolor[gray]{.95} Recruit size & DI & 921.31 &  1028.23 & 3378.43 & 4629.87 & 967.83 & 173.03 \\
\rowcolor[gray]{.95} & DD &              923.24 &  1026.63 & 3380.53 & 4631.84 & 969.06 & 175.02 \\ 
\rowcolor[gray]{.95} & $\Delta$AIC &     -1.93 &   1.61 &   -1.93 &   -1.97 &   -1.23 &  -1.99 \\
\hline
\end{tabular}
\end{spacing}
\end{table}

\begin{figure}[h!]
  \centering
  \includegraphics[width=\textwidth]{figures/densityDependenceFigure.pdf}
  \caption{\internallinenumbers (\textbf{A}) Within the same subpopulation, population growth rate (ln($\lambda$)) calculated from IPMs decreases as population size increases. (\textbf{B}) Within each subpopulation, population growth rate calculated by change in population size from year \textit{t} to year \textit{t+1} also decreases as population size increases. In \textbf{A} and \textbf{B}, each point represents values calculated from data from one transition in one subpopulation. Solid lines show linear regressions of the relationships between ln(N$_t$) and the respective response variable in each subpopulation. Numbers adjacent to these solid lines show the slope of each relationship, and are color-coded by subpopulation. Dashed grey lines show linear regressions of the relationships between ln(N$_t$) and the respective response variable across all subpopulations.}
  \label{fig:DDFig}
\end{figure} 


\textit{Vital Rate Buffering:} We did not find evidence of vital rate buffering in the \textit{O. coloradensis} populations we observed. Vital rate importance (either logistic VSS or elasticity) and variability (corrected SD) were not significantly negatively correlated, regardless of the simulated standard deviation for discrete vital rates we used(Fig. \ref{fig:VRBuffering}; correlation with minimum discrete vital rate SD (\textbf{A}): \textit{r} = 0.43 , \textit{P} = 0.25; correlation with maximum discrete vital rate SD (\textbf{B}): \textit{r} = -0.07 , \textit{P} = 0.85). As a vital rate became more important for determining population growth rate, it did not become significantly less variable, showing no evidence that vital rate buffering is taking place (Fig. \ref{fig:conceptualFigure}).  

\begin{figure}[h!]
  \centering
  \includegraphics[width=.9\textwidth]{figures/VRbufferingFigure.pdf}
  \caption{ \internallinenumbers The relationship between the variability of each vital rate (measured by corrected standard deviation) and its importance (measured by logit VSS or elasticity) does not show support for vital rate buffering. In these figures, a triangle indicates importance and variability for a discrete vital rate parameter, while a circle indicates the mean of importance and variability across an entire continuous vital rate function. Error bars around continuous vital rate means span the the 5$^{th}$ to 95$^{th}$ percentiles of either importance or variability values calculated for an entire continuous vital rate function. Dashed lines show the correlation between (mean) variability and (mean) importance across all vital rates. Because we lacked data to calculate the actual standard deviation of discrete vital rates, we simulated both the minimum and maximum possible standard deviation for each of these rates. (\textbf{A}) With the minimum possible discrete vital rate variability, there is a positive but insignificant correlation between vital rate variability and importance ( \textit{r} = 0.43 , \textit{P} = 0.25). (\textbf{B}) Using the maximum possible discrete vital rate variability, there is a negative but insignificant correlation between vital rate variability and importance (\textit{r} = -0.07 , \textit{P} = 0.85).}
  \label{fig:VRBuffering}
\end{figure} 

\textit{Asynchronous Responses and Source-Sink Dynamics:} We did not identify a signature of asynchronous responses to environmental variation in \textit{O. coloradensis} populations. There was not a significant relationship between the Spearman correlation of ln($\lambda$) between subpopulations and their spatial proximity (Mantel statistic = 0.396, \textit{P} = 0.06). We also performed Mantel tests using ln($\lambda$) correlation and distance matrices calculated uniquely for each population. There was not a significant relationship between subpopulation growth rate and spatial proximity at either Soapstone prairie (Mantel statistic = -0.659, \textit{P} = 0.83) or FEWAFB (Mantel statistic = 0.798, \textit{P} = 0.33). While these tests did not identify significant relationships, we did find a positive relationship between correlation of ln($\lambda$) and distance between subpopulations at Soapstone prairie, and a negative relationship between subpopulations at FEWAFB. Collectively, these results fail to provide support for both asynchronous responses and fine-scale source-sink dynamics in these \textit{O. coloradensis} populations. 

\begin{table}[h!]
\centering
\begin{spacing}{1.2}
\caption{\internallinenumbers Pearson correlations between mean growing season temperature coefficients in each continuous vital rate function. Below each correlation value is the \textit{P} value for that correlation. Bold text indicates a significant correlation. \label{Table:DemoComp}}
\begin{tabular}{p{.03\textwidth} c p{.1\textwidth} p{.1\textwidth} p{.1\textwidth} p{.1\textwidth} p{.1\textwidth}}
\toprule
& & \multicolumn{2}{c}{\textbf{Vital Rate}} \\ 
\cline{3-7}
& & \textit{Flowering} & \textit{Survival} & \textit{Growth} & \textit{Seed Prod.} & \textit{Recruit Size} \\ 
\hline
\multirow{5}{*}{\rotatebox{90}{\textbf{Vital Rate} }} & \cellcolor[gray]{.95}\textit{Flowering} & \cellcolor[gray]{.95}1.00 \:\:\: (0) & \cellcolor[gray]{.95}0.474 (0.342) & \cellcolor[gray]{.95}0.136 (0.797) & \cellcolor[gray]{.95}-0.073 (0.890) & \cellcolor[gray]{.95}-0.786 \: (0.064) \\
& \multicolumn{2}{r}{\textit{Survival}} &  1.00 \: \: (0) & 0.886 \textbf{(0.019)} & 0.675 (0.141) & -0.3570 \:(0.237) \\
 &  \multicolumn{3}{r}{\textit{\cellcolor[gray]{.95} Growth}} & \cellcolor[gray]{.95} 1.00 \: \: (0) & \cellcolor[gray]{.95} 0.664 (0.150) & \cellcolor[gray]{.95} -0.270  \:  (0.606) \\
& \multicolumn{4}{r}{\textit{Seed Prod.}} & 1.00 \:\:\: (0) & -0.432 \: (0.393) \\
 & \multicolumn{5}{r}{\cellcolor[gray]{.95}\textit{Recruit Size}} & \cellcolor[gray]{.95} 1.00 \:\:\:\:\: (0)\\
\hline
\end{tabular}
\end{spacing}
\end{table}


\section{Discussion}
\normalfont Our demographic analysis of the two largest known populations of the globally rare \textit{Oenothera coloradensis} evaluated the importance of seed banks to population dynamics and the demographic mechanisms that allow this rare species to persist. We found that including information about cryptic life stages alters the outcomes of the population model \cite{Paniw2017, Nguyen2019ConsequencesModels}, and that \textit{O. coloradensis} populations show signs of negative density-dependence at the subpopulation scale (Fig. \ref{fig:DDFig}; Table \ref{Table:DDModResults}). However, these populations do not show substantial evidence of demographic compensation, vital rate buffering, spatial asynchrony, or fine-scale source-sink dynamics. This may indicate that while these mechanisms may be important for the persistence of many small populations of rare plants, they are not strictly necessary in all cases. 

Including a discrete seed bank state in an IPM increased the asymptotic population growth rate compared to an IPM with only a continuous, size-based state, although both growth rates were still positive (Table \ref{IPMsTable}: with seed bank: IPM “B”, ln($\lambda$) = 0.65; without seed bank: IPM “A”, ln($\lambda$) = 0.27). The importance of including the seed bank in the model was consistent with our expectations, and also aligns with the conventional notion that seed banks can act as buffers against stochastic causes of population decline. The discrete rates for the probability of persisting and transitioning out of the seed bank have high elasticity in the IPMs in which they are included, but not the highest elasticity of any vital rate(Fig. \ref{fig:IPMKernel} C). The rate at which seeds produced by adult plants in year \textit{t} go into the seed bank in year \textit{t+1} is the vital rate function with highest elasticity. Previous matrix population models of \textit{O. coloradensis} without a seed bank state that were constructed in the 1990s identified the emergence rate of new seedlings as the vital rate most important for determining ln($\lambda$) \cite{Floyd1998}. Our finding that seed bank state transitions are important for this species aligns with this previous result, since rate of seedling emergence is the above-ground plant vital rate that is closest to the seed bank in this plant’s life cycle. An important caveat to our comparison of models with and without seed bank stages is the fact that the seed bank vital rate parameters we used were inferred from laboratory tests of germination and viability rates, which may be imperfect representations of \textit{in-situ} rates of viability and germination. The annual rate of seed death (10\%) was inferred from an \textit{in-situ} study, but is likely imprecise because of low sample size. Regardless of these potential sources of error, our results reinforce the fact that the seed bank can be an important element of a perennial plant’s life cycle, and if possible, should be modeled explicitly based on \textit{in-situ} estimates of the probability of seeds going into, persisting in, and emerging from the seed bank.      

We found evidence that, of the five proposed demographic mechanisms of small population persistence, negative density dependence was the only one acting in these \textit{O. coloradensis} populations. Including population size in the previous year as a covariate in vital rate models typically improved model fit, suggesting that density dependence is an important driver of growth, survival, and reproduction (Table \ref{Table:DDModResults}). Within a single subpopulation, population growth rate and the ratio of population size in year \textit{t+1} to year \textit{t} was generally higher when population size in year \textit{t} was smaller (Fig. \ref{fig:DDFig}), which indicates that negative density dependence prevents subpopulations from crashing when their population size is very small.  However, this pattern of higher growth rate at low population sizes did not exist when considering all subpopulations together (Fig. \ref{fig:DDFig}). This could indicate that each subpopulation is close to its carrying capacity for \textit{O. coloradensis}. This may indicate that the number of individuals is close to carrying capacity in each subpopulation, and that growth rate increases when the population size in a given subpopulation is small in comparison to its subpopulation-specific carrying capacity. \textit{O. coloradensis} vital rates had correlated responses to variation in the abiotic environment (Table \ref{Table:DemoComp}), which is the inverse of what is expected if vital rate buffering is taking place. It is possible that a signal of vital rate buffering would appear if we considered different abiotic variables such as disturbance frequency, or had more data. Vital rate buffering also was not identified, either with the minimum or maximum possible simulated discrete vital rate variability (Fig. \ref{fig:VRBuffering}). Vital rates with higher variability (higher SD) did not have a significantly higher or lower importance for determining ln($\lambda$) in comparison to less variable vital rates. This indicates that vital rate buffering is not stabilizing ln($\lambda$) after abiotic or demographic perturbation. The evidence for spatial asynchrony and fine-scale source-sink dynamics was also not strong. Mantel tests did not identify a significant relationship between the correlation of ln($\lambda$) between subpopulations and their spatial proximity, but did identify non-significant relationships between ln($\lambda$) correlation and proximity. However, this relationship was positive in Soapstone prairie subpopulations and negative in FEWAFB subpopulations, which provides inconsistent support for these mechanisms.  

It is somewhat surprising that negative density dependence is the only mechanism of small population persistence that has significant support in \textit{O. coloradensis} populations, since multiple mechanisms have been identified in at least one other rare species \cite{Dibner2019}. It is possible that support for one or more of these persistence mechanisms could emerge if more information about abiotic variation across space and time and data from more annual transitions was available for analysis. One potential explanation is that, while this species is a globally rare endemic with isolated subpopulations, it often grows at high local density. This strategy, which Rabinowitz describes as “locally abundant in a specific habitat but restricted geographically,” may allow \textit{O. coloradensis} to bypass the problems that small populations typically face, such as genetic and demographic bottlenecks that make them susceptible to stochastic environmental variation \cite{Rabinowitz1981SevenRarity}. It has also been shown that rare species are more likely than common species to benefit from facilitative interspecific interactions \cite{Calatayud2020PositiveAssemblages}. \textit{O. coloradensis} may participate in facilitative interactions with other species that increase its probability of persistence, although determining this will require further, community-level analysis. When compared to studies of persistence mechanisms in other species, our results illustrate the concept that species can employ distinct strategies for persistence \cite{Rabinowitz1981SevenRarity}. While demographic strategies that help maintain persistence may be effective for some species, other species may employ different strategies. This further emphasizes the importance of carefully considering the specific population and its community dynamics when managing and conserving rare species.  

Our analysis of the population dynamics of \textit{Oenothera coloradensis} at two distinct locations shows that this species has a life cycle that is strongly driven by introduction and persistence of seeds into a seed bank. More broadly, we show that this rare endemic species shows signs of negative density dependence. Populations of \textit{O. coloradensis} may additionally be maintained via high local abundances that allow them to escape the challenges of small population size that rare species often face \cite{Rabinowitz1981SevenRarity}. These findings reinforce the importance of careful evaluation of the unique population dynamics of rare species to inform successful conservation and management.  

\bibliography{references}

\newpage

% restart figure numbering
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{section}{0}
\setcounter{page}{1}


\newpage

\renewcommand{\thepage}{S\arabic{page}}
\renewcommand{\thesection}{S\arabic{section}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\thefigure}{S\arabic{figure}}


\section*{Supplementary Materials}
\normalfont
\subsection{Species Information}
\textit{Oenothera coloradensis} seeds are contained within small, indehiscent capsules that contain 2-5 seeds each \cite{Burgess2005CapsuleColoradensis}. A single adult individual can produce $>$500 capsules. This species does not reproduce vegetatively, although seeds typically germinate near the base of the parent plant, which often results in dense patches of vegetative individuals \cite{Heidel202133-YearWyoming}. \textit{O. coloradensis} is pollinated primarily by hawkmoths (Krakos, pers. comm. to B. Heidel, 2013). Seed dispersers are unknown \cite{Floyd1998, Heidel202133-YearWyoming}. Previous work established that \textit{O. coloradensis} population growth rate is particularly impacted by recruitment of seedlings \cite{Floyd1998}. Recruitment increases when non-\textit{O. coloradensis} community biomass is removed, indicating that surrounding grasses and forbs outcompete or shade-out seedlings \cite{Munk2002RosetteSpecies}.
\nocite{krakosPersonalComm}

\textit{O. coloradensis} commonly co-occurs with \textit{Agrostis stolonifera}, \textit{Pascopyrum smithii}, \textit{Poa pratensis}, \textit{Glycyrrhiza lepidota}, \textit{Iris missouriensis}, \textit{Cirsium flodmanii}, and \textit{Grindelia squarrosa} (Endangered and Threatened Wildlife and Plants, 2000; Munk et al., 2002). Encroachment of woody shrubs such as \textit{Salix exigua} has been correlated with declining numbers in some populations \cite{Heidel202133-YearWyoming}.
\nocite{USFWS2019}

The Wyoming Natural Diversity Database (WYNDD) began a base-wide census of reproductive individuals in the FEWAFB population in 1986, and has repeated this census annually since 1988 \cite{Heidel202133-YearWyoming}. The first estimate of species size after its full geographic range was identified occurred in 1998, when it was approximated that the entire species consisted of 47,300 to 50,300 reproductive individuals \cite{Fertig2000-ow}.

\subsection{Discrete Vital Rate Parameter Estimation}
 Previously-published data from a greenhouse experiment using \textit{O. coloradensis} seed capsules collected from the FEWAFB populations determined that viable seeds had an average germination rate of 20.3\% after cold-stratification, and did not identify a consistent decline in germination rate over five years \cite{Burgess2005CapsuleColoradensis}. This study also found that 58.5\% of seeds produced were viable. We conducted an additional seed study to determine if overwintering in natural conditions lead to a lower germination rate than was identified in the previous greenhouse study. We buried 60 field-collected seed capsules in mesh bags at 6 locations near our demographic study plots at FEWAFB, and then recovered the seed bags after one winter. An average of 10\% of seed capsules were not recoverable, likely because they were non-viable and withered away or were eaten. We planted the recovered capsules in standard greenhouse conditions, and found a mean germination rate of 6.8\%. This germination rate was much lower than that identified by Burgess et al., however our seed study had a much smaller sample size, reducing the reliability of our result \cite{Burgess2005CapsuleColoradensis}. However, it is still likely that true germination rates are much lower than those identified in greenhouse conditions, so we reduced the germination rate identified by Burgess et al. by 20\%. 

\begin{table}[h!]
\centering
\begin{spacing}{1.2}
\caption{ \internallinenumbers Permanent Plot Locations and subpopulation-level sample sizes for each year and individual type (seedling vs. non-seedling). GPS coordinates listed in decimal degrees, map datum and spheroid: WGS 84.\label{plotLocationTable}}
\begin{tabular}{cc p{.06 \textwidth} p{.08 \textwidth} p{.11 \textwidth} |p{.03 \textwidth}p{.03 \textwidth}|p{.03 \textwidth}p{.03 \textwidth}|p{.03 \textwidth}p{.03 \textwidth}}
\toprule
 & & & & & \multicolumn{6}{c}{Sample Size}\\
 & & & & & \multicolumn{2}{c}{2018} & \multicolumn{2}{c}{2019} & 2020 \\ 
Site & Subpopulation & Plot Name & N \:\:\:\: Coord. & W  \:\:\:\:\:\:\:\: Coord. & \rotatebox{90}{non-seedling} & \rotatebox{90}{seedling} & \rotatebox{90}{non-seedling} & \rotatebox{90}{seedling} & \rotatebox{90}{non-seedling} & \rotatebox{90}{seedling}\\ 
\hline
\multirow{9}{*}{\rotatebox{90}{FEWAFB}}  & \cellcolor[gray]{.95} Unnamed Creek & \cellcolor[gray]{.95} U3 & \cellcolor[gray]{.95} \small 41.13642 & \cellcolor[gray]{.95} \small -104.87209 & \multirow{3}{*}{\small740} & \multirow{3}{*}{\small525} & \multirow{3}{*}{\small528} & \multirow{3}{*}{\small417} & \multirow{3}{*}{\small406}& \multirow{3}{*}{\small530} \\
 &Unnamed Creek & U4& \small 41.13634 & \small -104.87183 & & & & \\ 
 & \cellcolor[gray]{.95}Unnamed Creek & \cellcolor[gray]{.95}U6  & \cellcolor[gray]{.95}\small 41.13647 & \cellcolor[gray]{.95}\small -104.87132 & & & &  \\
\cline{2-11}
 & Diamond Creek & D7 & \small 41.14340 & \small -104.88380 & \multirow{3}{*}{\small235} & \multirow{3}{*}{\small209} & \multirow{3}{*}{\small347} & \multirow{3}{*}{\small149} & \multirow{3}{*}{\small275} & \multirow{3}{*}{\small81}\\
  & \cellcolor[gray]{.95} Diamond Creek & \cellcolor[gray]{.95} D10 & \cellcolor[gray]{.95} \small 41.14441& \cellcolor[gray]{.95} \small-104.88303 & &&& \\
 & Diamond Creek & D11 & \small 41.14431& \small -104.88094 &&&&\\
\cline{2-11}
  & \cellcolor[gray]{.95} Crow Creek & \cellcolor[gray]{.95}C4 & \cellcolor[gray]{.95}\small 41.15540 & \cellcolor[gray]{.95}\small -104.87497 & 
  \multirow{3}{*}{\small203} & \multirow{3}{*}{\small127} & \multirow{3}{*}{\small214} & \multirow{3}{*}{\small98} & \multirow{3}{*}{\small150} & \multirow{3}{*}{\small160}\\ 
 & Crow Creek & C5 & \small 41.15477 & \small -104.87474 &&&&\\
  & \cellcolor[gray]{.95}Crow Creek & \cellcolor[gray]{.95}C8 & \cellcolor[gray]{.95}\small 41.15534 & \cellcolor[gray]{.95}\small -104.87487 &&&&\\
\hline
 \multirow{9}{*}{\rotatebox{90}{Soapstone}} & Pasture HQ5 & S1 & \small 40.99297& \small -105.00925 & \multirow{3}{*}{\small283} & \multirow{3}{*}{\small772} & \multirow{3}{*}{\small714} & \multirow{3}{*}{\small813} & \multirow{3}{*}{\small641} & \multirow{3}{*}{\small423}\\
 & \cellcolor[gray]{.95}Pasture HQ5 & \cellcolor[gray]{.95}S2 & \cellcolor[gray]{.95}\small40.99318& \cellcolor[gray]{.95}\small-105.00935 &&&&\\
  & Pasture HQ5 & S3 & \small40.99342& \small-105.00937 &&&& \\
\cline{2-11}
  & \cellcolor[gray]{.95}Pasture HQ3 & \cellcolor[gray]{.95}S4 & \cellcolor[gray]{.95}\small40.98623& \cellcolor[gray]{.95}\small-105.01691 & \multirow{3}{*}{\small102} & \multirow{3}{*}{\small138} & \multirow{3}{*}{\small158} & \multirow{3}{*}{\small173} & \multirow{3}{*}{\small117} & \multirow{3}{*}{\small104}\\
  & Pasture HQ3 & S5 & \small40.98639& \small-105.01671 &&&&\\
 & \cellcolor[gray]{.95}Pasture HQ3 & \cellcolor[gray]{.95}S6 & \cellcolor[gray]{.95}\small40.98650& \cellcolor[gray]{.95}\small-105.01656 &&&&\\
\cline{2-11}
  & Meadow & S7 & \small40.98753 & \small-105.02148 & \multirow{3}{*}{\small44} & \multirow{3}{*}{\small31} & \multirow{3}{*}{\small47} & \multirow{3}{*}{\small28} & \multirow{3}{*}{\small48} & \multirow{3}{*}{\small12}\\
  & \cellcolor[gray]{.95} Meadow & \cellcolor[gray]{.95} S8 & \cellcolor[gray]{.95} \small40.98747 & \cellcolor[gray]{.95} \small-105.02179 &&&&\\
  & Meadow & S9 & \small40.98724 & \small-105.02145 &&&&\\
\hline
\end{tabular}
\end{spacing}
\end{table}

\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/COBP_mapFigure.pdf}
    \caption{\internallinenumbers(\textbf{A}) The current known distribution of \textit{O. coloradensis}, shown in dark blue, extends into Wyoming, Colorado, and Nebraska. The historical distribution included the current distribution area as well as some additional locations shown in pale blue. Distribution information comes from Everson, 2019. Black dots show the relative locatino of the FEWAFB and Soapstone prairie populations included in this study. Colored dots show the location of plots in each subpopulation at FEWAFB (\textbf{B}) and Soapstone Prairie (\textbf{C}).}
    \label{fig:plotMap}
\end{figure}

\begin{figure}[ht!]
  \centering
  \includegraphics[width=.7\textwidth]{seedRegressionPlot.pdf}
  \caption{ \internallinenumbers As the stem length of an \textit{Oenothera coloradensis} flowering individual increases, the number of capsules it produces increases as well. The red line shows the fit from a Poisson generalized linear model, and the grey ribbon shows the 95\% confidence interval around the fitted relationship. Model equation: Number of capsules = $e^{(1.843 + 0.119\text{\textsf{x}}\text{S}))}$, where \textit{S} is stem length in cm (pseudo R-squared = 0.42, \textit{P} = $<$ 0.01, Residual deviance = 186.98, df = 104).  }
  \label{fig:seedRegression}
\end{figure} 


\begin{spacing}{1.3}
    \begin{longtable}[ht!]{c|l|p{0.73\textwidth}}
    \caption{Continuous vital rate functions used in each IPM. In the functions below, $size_t$ indicates plant longest leaf length in the previous year, $N_t$ indicates number of individuals in a plot in the previous year, $T_G$ indicates mean temperature in the previous year's growing season, and $T_W$ indicates the mean temperature in winter of the previous year. 
    \label{tab:VitalRateFunctions}} \\
    \rowcolor[gray]{.95} IPM & Vital Rate & Equation  \\
        \hline
        A \& B & Survival  &   $logit(\textit{s(z)}) = -0.21 + 0.37 (ln(size_t)) $ \\
         \rowcolor[gray]{.95}& Flowering  & logit(\textit{Pb(z)})  $= -34.06 + 27.67(ln(size_t)) -5.78 (ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.57$ + $0.18 (ln(size_t))$; $\sigma_s = 0.51$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 3.35 + 1.19 (ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.21$; $\sigma_r = 0.77$ \\
         \hline
        \rowcolor[gray]{.95}C &   Survival  &   $logit(\textit{s(z)}) = -0.73 + 0.45 (ln(size_t)) $ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -40.31 + 33.40(ln(size_t)) -7.09 (ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.64$ + $0.11 (ln(size_t))$; $\sigma_s = 0.42$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.73 + 1.01 (ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.19$; $\sigma_r = 0.76$ \\
         \hline
        D &  Survival  &   $logit(\textit{s(z)}) = 1.01 + -0.01 (ln(size_t)) $ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -44.52 + 36.73(ln(size_t)) -7.69 (ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.84 + 0.26(ln(size_t))$; $\sigma_s = 0.52$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 3.80 + 1.11 (ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.22$; $\sigma_r = 0.78$ \\
         \hline
        \rowcolor[gray]{.95}E &   Survival  &   $logit(\textit{s(z)})= 0.21 +0.18(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -18.73 + 15.84(ln(size_t)) -3.63 (ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.74 + 0.24(ln(size_t))$; $\sigma_s = 0.49$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.83 + 0.92 (ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.19$; $\sigma_r = 0.80$ \\
         \hline
        F &  Survival  &   $logit(\textit{s(z)})= 0.54 +0.24(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -27.49 + 20.83(ln(size_t)) -4.04 (ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.62 + 0.16(ln(size_t))$; $\sigma_s = 0.44$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 1.72 + 1.51 (ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.15$; $\sigma_r = 0.80$ \\
         \hline
        \rowcolor[gray]{.95}G &  Survival  &  $logit(\textit{s(z)})= -0.19 +0.30(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -32.72 + 23.50(ln(size_t)) -4.23 (ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.42 + 0.21(ln(size_t))$; $\sigma_s = 0.42$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 0.83 + 2.06 (ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.21$; $\sigma_r = 0.77$ \\
         \hline
        H &  Survival  &  $logit(\textit{s(z)})= -0.32 +0.62(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -31.04 + 23.96(ln(size_t)) -4.71 (ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.50 + 0.05(ln(size_t))$; $\sigma_s = 0.43$\\
        \rowcolor[gray]{.95} & Seed prod.  & $exp(\textit{b(z)}) = 2.53 + 1.59 (ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.23$; $\sigma_r = 0.76$ \\
         \hline
        \rowcolor[gray]{.95}I &  Survival  &  $logit(\textit{s(z)})= -0.33 +0.46(ln(size_t)) - 0.0003 (N_t)$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -41.36 + 33.73(ln(size_t)) -7.21 (ln(size_t)^2) + 0.0008(N_t)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 2.19 + 0.14(ln(size_t)) - 0.0005(N_t)$; $\sigma_s = 0.42$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.73 + 1.01(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.19$; $\sigma_r = 0.76$ \\
         \hline
         J &  Survival  &  $logit(\textit{s(z)})= 17.59 +0.05(ln(size_t)) - 0.03 (N_t)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -46.27 + 37.38(ln(size_t)) -7.83 (ln(size_t)^2) + 0.002(N_t)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 3.70 + 0.28(ln(size_t)) - 0.004(N_t)$; $\sigma_s = 0.51$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 3.80 + 1.11(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.22$; $\sigma_r = 0.78$ \\
         \hline
         K\rowcolor[gray]{.95} &  Survival  &  $logit(\textit{s(z)})= -13.85 +0.20(ln(size_t)) + 0.04(N_t)$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -21.21 + 15.50(ln(size_t)) -3.55(ln(size_t)^2) + 0.009(N_t)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = -0.49 + 0.24(ln(size_t)) + 0.006(N_t)$; $\sigma_s = 0.48$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.83 + 0.92(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.19$; $\sigma_r = 0.80$ \\
         \hline
         L  &  Survival  &  $logit(\textit{s(z)})= 33.06 + 0.24(ln(size_t)) -0.44(N_t)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -26.23 + 21.0(ln(size_t)) -4.01(ln(size_t)^2) -0.02(N_t)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 15.89 + 0.17(ln(size_t)) -0.19(N_t)$; $\sigma_s = 0.43$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 1.72 + 1.51(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.15$; $\sigma_r = 0.80$ \\
         \hline
         \rowcolor[gray]{.95}M  &  Survival  &  $logit(\textit{s(z)})= 5.39 + 0.37(ln(size_t)) -0.02(N_t)$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -32.79 + 24.02(ln(size_t)) -4.34(ln(size_t)^2) -0.002(N_t)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 2.26 + 0.23(ln(size_t)) - 0.003(N_t)$; $\sigma_s = 0.39$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 0.83 + 2.06(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.21$; $\sigma_r = 0.77$ \\
         \hline
         N  &  Survival  &  $logit(\textit{s(z)})= 4.30 + 0.87(ln(size_t)) -0.004(N_t)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -28.50 + 25.52(ln(size_t)) -4.93(ln(size_t)^2) -0.004(N_t)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 2.76 + 0.16(ln(size_t)) - 0.001(N_t)$; $\sigma_s = 0.36$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 2.53 + 1.59(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.23$; $\sigma_r = 0.77$ \\
         \hline
         \rowcolor[gray]{.95}S  &  Survival  &  $logit(\textit{s(z)})= 1.11 + 0.49(ln(size_t)) + 0.005(N_t) - 0.28 (T_G)$ \\
         & Flowering  & $logit(\textit{Pb(z)}) = -47.12 + 33.70(ln(size_t)) -7.21(ln(size_t)^2) +0.0004 (N_t) + 0.42 (T_G) - 0.23 (T_W)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 4.12 + 0.14(ln(size_t)) +0.0009(N_t) -0.20 (T_G)$; $\sigma_s = 0.41$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.52 + 0.90(ln(size_t)) + 0.0008 (N_t) + 0.01 (T_G) + 0.10 (T_W)$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = -0.38 + 0.00005 (N_t) + 0.04 (T_G) + 0.03 (T_W) $; $\sigma_r = 0.76$ \\
         \hline
         T  &  Survival  &  $logit(\textit{s(z)})= -27.59 + 0.21(ln(size_t)) -0.0006(N_t) + 1.91 (T_G)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -70.28 + 35.37(ln(size_t)) -7.45(ln(size_t)^2) - 0.003 (N_t) + 1.83 (T_G) - 0.95 (T_W)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = -2.88 + 0.27(ln(size_t)) +0.0002 (N_t) + 0.32 (T_G)$; $\sigma_s = 0.49$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = -12.0 + 0.80(ln(size_t)) -0.002 (N_t) + 1.07 (T_G) - 0.94 (T_W)$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = -0.65 -0.0007 (N_t) + 0.06 (T_G) - 0.16 (T_W) $; $\sigma_r = 0.77$ \\
         \hline
         \rowcolor[gray]{.95}U  &  Survival  &  $logit(\textit{s(z)})= -14.02 + 0.39(ln(size_t)) + 0.006(N_t) + 0.87 (T_G)$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -22.83 + 16.0(ln(size_t)) -3.66(ln(size_t)^2) + 0.002 (N_t) + 0.19 (T_G) - 1.01 (T_W)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 0.40 + 0.25(ln(size_t)) - 0.001 (N_t) + 0.10 (T_G)$; $\sigma_s = 0.46$\\
         & Seed prod.  & $exp(\textit{b(z)}) = -14.33 + 0.59(ln(size_t)) + 0.001 (N_t) + 1.18 (T_G) - 1.17 (T_W)$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = -0.56 -0.00002 (N_t) + 0.05 (T_G) + 0.09 (T_W) $; $\sigma_r = 0.81$ \\
         \hline
         V  &  Survival  &  $logit(\textit{s(z)})= -6.78 + 0.24(ln(size_t)) -0.08(N_t) + 0.69 (T_G)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -36.50 + 18.80(ln(size_t)) -3.57(ln(size_t)^2)-0.02 (N_t) + 0.79 (T_G) +0.09 (T_W)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = -1.83 + 0.17(ln(size_t)) + 0.0003 (N_t) + 0.23 (T_G)$; $\sigma_s = 0.42$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 8.44 + 1.32(ln(size_t)) + 0.003 (N_t) -0.43 (T_G) - 0.18 (T_W)$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = -2.01 + 0.006 (N_t) + 0.13 (T_G) -0.01 (T_W) $; $\sigma_r = 0.80$ \\
         \hline
         \rowcolor[gray]{.95}W  &  Survival  &  $logit(\textit{s(z)})= -32.30 + 0.38(ln(size_t)) +0.0008(N_t) + 2.19(T_G)$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -311.80 + 18.72(ln(size_t)) -3.56(ln(size_t)^2)-0.002 (N_t) + 17.93 (T_G) -20.46 (T_W)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = -2.73 + 0.22(ln(size_t)) -0.0006 (N_t) + 0.29 (T_G)$; $\sigma_s = 0.39$\\
         & Seed prod.  & $exp(\textit{b(z)}) = -7.07 + 1.47(ln(size_t)) + 0.003 (N_t) +0.63(T_G)$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 1.75 + 0.006 (N_t) - 0.10 (T_G) + 0.17 (T_W) $; $\sigma_r = 0.77$ \\
         \hline
         X  &  Survival  &  $logit(\textit{s(z)})= -34.59 + 0.88(ln(size_t)) +0.001(N_t) + 2.26(T_G)$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -43.64 + 28.21(ln(size_t)) -5.35(ln(size_t)^2)-0.001 (N_t) + 0.57 (T_G) +1.33 (T_W)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = -7.65 + 0.16(ln(size_t)) -0.0000004 (N_t) + 0.61 (T_G)$; $\sigma_s = 0.36$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = -28.65 + 0.62(ln(size_t)) + 0.0005 (N_t) +2.15(T_G) - 1.64 (T_W)$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.48 - 0.00004 (N_t) - 0.01 (T_G) + 0.005 (T_W) $; $\sigma_r = 0.77$ \\
         \hline
         \rowcolor[gray]{.95}AA  &  Survival  &  $logit(\textit{s(z)})= -0.26 + 0.54(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -28.96 + 21.71(ln(size_t)) -4.14(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.48 + 0.10(ln(size_t))$; $\sigma_s = 0.44$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.12 + 1.24(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.23$; $\sigma_r = 0.77$ \\
         \hline
         BB  &  Survival  &  $logit(\textit{s(z)})= -0.18 + 0.29(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -32.70 + 26.93(ln(size_t)) -5.73(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.72 + 0.18(ln(size_t))$; $\sigma_s = 0.51$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 3.45 + 1.17(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.19$; $\sigma_r = 0.77$ \\
         \hline
         \rowcolor[gray]{.95}CC  &  Survival  &  $logit(\textit{s(z)})= -1.23 + 0.76(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -45.81 + 38.00(ln(size_t)) -8.03(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.57 + 0.12(ln(size_t))$; $\sigma_s = 0.44$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.21 + 1.22(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.13$; $\sigma_r = 0.77$ \\
         \hline
         DD  &  Survival  &  $logit(\textit{s(z)})= 2.43-0.23(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -86.83 + 71.52(ln(size_t)) -14.65(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.93 + 0.25(ln(size_t))$; $\sigma_s = 0.56$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 6.01 + 0.34(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.30$; $\sigma_r = 0.69$ \\
         \hline
         \rowcolor[gray]{.95}EE  &  Survival  &  $logit(\textit{s(z)})= 0.65+0.15(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -24.35 + 17.91(ln(size_t)) -3.49(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.85 + 0.20(ln(size_t))$; $\sigma_s = 0.52$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 2.51 + 1.54(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.09$; $\sigma_r = 0.86$ \\
         \hline
         FF  &  Survival  &  $logit(\textit{s(z)})= 1.50-0.32(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -70.62 + 54.30(ln(size_t)) -10.46(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.65 + 0.21(ln(size_t))$; $\sigma_s = 0.46$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 1.91 + 1.39(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.10$; $\sigma_r = 0.92$ \\
        \hline
         \rowcolor[gray]{.95}GG  &  Survival  &  $logit(\textit{s(z)})= 0.99+0.12(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -50.32 + 36.50(ln(size_t)) -6.54(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.48 + 0.28(ln(size_t))$; $\sigma_s = 0.38$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 3.06 + 1.22(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.18$; $\sigma_r = 0.81$ \\ \hline
         HH  &  Survival  &  $logit(\textit{s(z)})= 0.74+0.33(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -50.91 + 40.17(ln(size_t)) -7.81(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.62 + 0.18(ln(size_t))$; $\sigma_s = 0.39$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 4.20 + 0.97(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.24$; $\sigma_r = 0.76$ \\ 
         \hline
         \rowcolor[gray]{.95}II  &  Survival  &  $logit(\textit{s(z)})= -0.26+0.06(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -41.11 + 36.347(ln(size_t)) -8.31(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.67 + 0.16(ln(size_t))$; $\sigma_s = 0.38$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 4.92 + 0.41(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.22$; $\sigma_r = 0.71$ \\
         \hline
         JJ  &  Survival  &  $logit(\textit{s(z)})= 0.07+0.16(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -24.83 + 19.82(ln(size_t)) -4.20(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.63 + 0.32(ln(size_t))$; $\sigma_s = 0.44$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 4.63 + 0.66(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.09$; $\sigma_r = 0.82$ \\
         \hline
         \rowcolor[gray]{.95}KK  &  Survival  &  $logit(\textit{s(z)})= -0.26+0.25(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -27.90 + 27.66(ln(size_t)) -7.00(ln(size_t)^2)$ \\
        \rowcolor[gray]{.95} & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.56 + 0.30(ln(size_t))$; $\sigma_s = 0.41$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 5.72 -0.09(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.80$; $\sigma_r = 0.79$ \\
         \hline
         LL  &  Survival  &  $logit(\textit{s(z)})= -0.22+0.77(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -147.87 + 121.01(ln(size_t)) -24.82(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.61 + 0.10(ln(size_t))$; $\sigma_s = 0.39$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) = 1.98 + 1.57(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.16$; $\sigma_r = 0.68$ \\
         \hline
         \rowcolor[gray]{.95}MM  &  Survival  &  $logit(\textit{s(z)})= -1.13+0.60(ln(size_t))$ \\
         & Flowering  & $logit(\textit{Pb(z)})  = -45.56 + 39.31(ln(size_t)) -8.64(ln(size_t)^2)$ \\
         \rowcolor[gray]{.95}& Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.34 + 0.14(ln(size_t))$; $\sigma_s = 0.40$\\
         & Seed prod.  & $exp(\textit{b(z)}) = 0.71 + 2.00(ln(size_t))$  \\
         \rowcolor[gray]{.95}& Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.32$; $\sigma_r = 0.64$ \\
         \hline
         NN  &  Survival  &  $logit(\textit{s(z)})= -1.74+1.39(ln(size_t))$ \\
         \rowcolor[gray]{.95}& Flowering  & $logit(\textit{Pb(z)})  = -27.89 + 20.50(ln(size_t)) -3.99(ln(size_t)^2)$ \\
         & Growth  &  $G(z',z) = N(\mu_s, \sigma_s)$; $\mu_s = 1.17 + 0.13(ln(size_t))$; $\sigma_s = 0.33$\\
         \rowcolor[gray]{.95}& Seed prod.  & $exp(\textit{b(z)}) =  5.49+ 0.14(ln(size_t))$  \\
         & Recruit size  &  $c_o(z') = N(\mu_r, \sigma_r)$; $\mu_r = 0.24$; $\sigma_r = 0.78$ \\
         \end{longtable}
\end{spacing}


\end{document}